sys-devel/dwz/dwz-0.14.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
You should use our automated tools to commit so it fixes the copyright header automatically. E.g. repoman or pkgdev.